library/std/src/fs/tests.rs

```diff
@@ -192,6 +193,7 @@ fn file_test_io_seek_and_tell_smoke_test() {
         tell_pos_post_read = check!(read_stream.seek(SeekFrom::Current(0)));
     }
     check!(fs::remove_file(filename));
+
```

Comment:
This seems to be unrelated. Can you remove the added lines?

Reply:
I deleted the unused line.